frontend/leaderboard/src/Bindings.re

```diff
@@ -24,6 +26,15 @@ module GoogleSheets = {
     range: string,
   };
 
+  type sheetsQueryData = {values: array(array(string))};
```

Comment:
Just noticed that `sheetsQueryData` is a typo and should be renamed `sheetsUpdateData`

Reply:
If they're the same, maybe there can just be "sheetData" that is used in both cases